engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/SignalDto.ftl

```diff
@@ -0,0 +1,42 @@
+{
+  "type" : "object",
+  "properties" : {
+
+    <@lib.property
+        name = "name"
+        type = "string"
+        desc = "The name of the signal to deliver.
+                Note: This property is mandatory."/>
+
+    <@lib.property
+        name = "executionId"
+        type = "string"
+        desc = "Optionally specifies a single execution which is notified by the signal.
+                Note: If no execution id is defined the signal is broadcasted to all subscribed handlers."/>
+
+    <@lib.property
+        name = "variables"
+        type = "object"
+        additionalProperties = true
+        dto = "VariableValueDto"
+        desc = "A JSON object containing variable key-value pairs. Each key is a variable name and each
+                value a JSON variable value object."/>
+
+    <@lib.property
+        name = "tenantId"
+        type = "string"
+        desc = "Specifies a tenant to deliver the signal. The signal can only be received on executions
+                or process definitions which belongs to the given tenant.
+
+                Note: Cannot be used in combination with executionId."/>
+
+    <@lib.property
+        name = "withoutTenantId"
+        type = "boolean"
```

Comment:
We should specify that the `defaultValue` is `false`.
```suggestion
        type = "boolean"
        defaultValue = false
```

Reply:
@mboskamp, we had a discussion with @yanavasileva regarding default values of `boolean` properties. We should be careful when we set default values. For this one specifically, I made a mistake and you can ignore the above suggestion. The docs clearly state that the value **may not be** `false`. If we see that the docs explicitly define a default value, then we can set it.